addons/cartel/vehicles/CfgCommandoVehicles.hpp

```diff
@@ -0,0 +1,58 @@
+// commando
+class CUP_I_LR_SF_HMG_AAF;
+class GVAR(commando_LR_MG): CUP_I_LR_SF_HMG_AAF {
+    side = 0;
+    faction = QUOTE(ADDON);
+    displayName = "Commando Land Rover MG";
+    hiddenSelectionsTextures[] = { "cup\wheeledvehicles\cup_wheeledvehicles_lr\data\textures\civ_r_lr_base_co.paa","cup\wheeledvehicles\cup_wheeledvehicles_lr\data\textures\civ_r_lr_special_co.paa","cup\wheeledvehicles\cup_wheeledvehicles_lr\data\jackal_base_woodlnd_co.paa","cup\wheeledvehicles\cup_wheeledvehicles_lr\data\jackal_mount_woodlnd_co.paa" };
+    crew = QGVAR(commando_hk416);
+    typicalCargo[] = {
+        QGVAR(commando_hk416);
+    };
```

Comment:
```suggestion
    };
    INVENTORY_EMPTY;
```

Add `INVENTORY_EMPTY` to all of the vehicles unless you need items in them